packages/svelte/src/index.d.ts

```diff
@@ -320,19 +320,21 @@ export type MountOptions<Props extends Record<string, any> = Record<string, any>
 	 */
 	target: Document | Element | ShadowRoot;
 	/**
-	 * Optional node inside `target` and when specified, it is used to render the component immediately before it.
+	 * Optional node inside `target`. When specified, it is used to render the component immediately before it.
 	 */
 	anchor?: Node;
 	/**
 	 * Allows the specification of events.
+	 * @deprecated Use callback props instead.
 	 */
 	events?: Record<string, (e: any) => any>;
 	/**
-	 * Used to define context at the component level.
+	 * Context that can be access via `getContext()` at the component level.
```

Comment:
don't think you need to lead off with saying it's "Context" since that's the variable name and you say it's returned from `getContext`
```suggestion
	 * Can be accessed via `getContext()` at the component level.
```